### ISSUE TITLE
Add grouping for tasks in AzureDevOps pipeline 

### ISSUE THREAD

**Issue by JuergenRB** [cake-contrib/Cake.BuildSystems.Module#111]
With the changes of #110 it is possible to use the ##[group] command.

This can be used to change the output from 
![image](https://user-images.githubusercontent.com/22403655/147575571-d029e2cf-8e54-44d5-b88e-4dbfbf55f0aa.png)
to
![image](https://user-images.githubusercontent.com/22403655/147575596-c7096057-168c-4a6f-a344-5660f811b3c4.png)

**Comment by JuergenRB** [comment]
I made the changes in the [Fork-Branch](https://github.com/JuergenRB/Cake.BuildSystems.Module/tree/feature/AddAzurePipelinesExecutionStrategyForGrouping) and would make a new PR after #110
